src/stories/4-elements/accordion/accordion.stories.tsx

```diff
@@ -0,0 +1,243 @@
+import { action } from "@storybook/addon-actions";
+import { ArgsTable, Description, Heading, PRIMARY_STORY, Stories, Title } from "@storybook/addon-docs";
+import { Meta, StoryFn } from "@storybook/react";
+import { useEffect, useRef } from "react";
+import { IFrontendEngineRef } from "../../../components";
+import { IAccordionSchema } from "../../../components/elements/accordion/types";
+import { CommonFieldStoryProps, DefaultStoryTemplate, FrontendEngine, SUBMIT_BUTTON_SCHEMA } from "../../common";
+
+const meta: Meta = {
+	title: "Element/Accordion",
+	parameters: {
+		docs: {
+			page: () => (
+				<>
+					<Title>Accordion</Title>
+					<Description>
+						Wrapping component that must be rendered as a direct descendant of `accordion` uiType.
+					</Description>
+					<Heading>Props</Heading>
+					<ArgsTable story={PRIMARY_STORY} />
+					<Stories includePrimary={true} title="Example" />
+				</>
+			),
+		},
+	},
+	argTypes: {
+		...CommonFieldStoryProps("accordion", true),
+		children: {
+			description: "Elements that are the descendant of this component",
+			table: {
+				type: { summary: "TFrontendEngineFieldSchema" },
+			},
+			type: { name: "object", value: {}, required: true },
+		},
+		button: {
+			description:
+				"<div>Define button-related settings.</div><ul><li><strong>label</strong>: Label of the button (Default: Edit)</li></ul>",
+			table: {
+				type: {
+					summary: "{ label: string }",
+				},
+				defaultValue: { summary: "{ label: 'Edit' }" },
```

Comment:
can also indicate to use `false` to hide the button

```suggestion
				defaultValue: { summary: "boolean | { label: 'Edit' }" },
```